xbmc/playlists/PlayListTypes.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+namespace PLAYLIST
+{
+
+using Id = int;
+
+constexpr Id TYPE_NONE = -1; //! Playlist id of type none
```

Comment:
Can you remember us what TYPE_NONE actually is? IIRC correctly it simple means we don't know the type, i.e. the playlist type is undefined right?
If so maybe `TYPE_UNDEFINED` conveys better semantics than `TYPE_NONE` since a playlist is always something tangible/concrete.

Reply:
im not sure if is good rename as undefined
at least looking at code seem to means that no playlist is used

Reply:
yeap figured that too. Also you return a playlist_id, so NONE makes sense.